ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterSpectator.java

```diff
@@ -63,15 +63,17 @@ public void spectate() throws Exception {
     // zk connection fails on both data centers, then things like replication between data centers might just stop.
     // For now, since we have only one fabric in cloud, and the spectator is being used for only cloud to store replication, this will work.
     // Once we add more fabrics, we should revisit this.
-    for (Map.Entry<String, DcZkInfo> entry : dataCenterToZkAddress.entrySet()) {
-      String zkConnectStr = entry.getValue().getZkConnectStr();
-      HelixManager helixManager =
-          helixFactory.getZKHelixManager(cloudConfig.vcrClusterName, selfInstanceName, InstanceType.SPECTATOR,
-              zkConnectStr);
-      helixManager.connect();
+    for (DcZkInfo dcZkInfo: dataCenterToZkAddress.values()) {
+      // only handle vcr clusters for now
+      if (dcZkInfo.getReplicaType() == ReplicaType.CLOUD_BACKED) {
+        HelixManager helixManager =
+            helixFactory.getZKHelixManager(cloudConfig.vcrClusterName, selfInstanceName, InstanceType.SPECTATOR,
+                dcZkInfo.getZkConnectStr());
```

Comment:
Does `dcZkInfo.getZkConnectStr()` return empty string for now?

Reply:
No, this would be a case where it would be set to the vcr cluster address. Hopefully, we can find a way to unify `HelixClusterSpectator` with the rest of the `ClusterMap` eventually.